src/transformers/models/deta/modeling_deta.py

```diff
@@ -2228,17 +2247,13 @@ def forward(self, outputs, targets):
             enc_outputs = outputs["enc_outputs"]
             bin_targets = copy.deepcopy(targets)
             for bt in bin_targets:
-                bt["labels"] = torch.zeros_like(bt["labels"])
+                bt["class_labels"] = torch.zeros_like(bt["class_labels"])
```

Comment:
Any reason this is renamed to `class_labels`?

Reply:
@NielsRogge targets has no longer have dict keys with "labels" they only contain "class_labels" in DETR.

Reply:
Thanks, not sure why this wasn't caught!

Reply:
@NielsRogge Because original code was not having 'enc_ouput'. See line 1956

Reply:
@amyeroberts bin_targets which is groundtruth does not have the label called "labels". In addition previously used model was not passing this line of code.

Reply:
Ok, fine for me